### ISSUE TITLE
Suggestion: Bare Bones Version?

### ISSUE THREAD

**Issue by Avery93** [Ivorforce/RecurrentComplex#382]
I love the more lived-in feel these structures give to the world on generation, but I don't need the utility of spawning more post-generation, and I would love for this to be compatible with other decoration replacement mods like Dynamic Trees. See issue #349, I'm having the same problem despite turning off nature spawning, turning off all associated structures manually in the #structure tweaks GUI, AND blacklisting the roofed forest biome in the config file. 

I'm trying to set up an adventure modpack for personal use, and if this were to just add new structures to natural generation without touching the trees, etc. I'd be thrilled to death, as no other structure mods have a default repository that suits the feel of vanilla minecraft so well.

**Comment by CREEATION** [comment]
`but I don't need the utility of spawning more post-generation`

If you are talking about "making more structures", there is a config option to make Recurrent Complex "server-side only":

```
    # Enabling this will make the mod register as little as possible, which enables it to be used server-side only. Note that this prevents you from editing or importing structures. [default: false]
    B:lightweightMode=false
```

**Comment by Avery93** [comment]
That is definitely a config I would be interested in changing, but ultimately that's not the core of what I want. I was saying that I'd like a version that simply adds more structures without changing anything at all about vanilla. The core of the issue isn't really dynamic trees compatibility, but that what could be a lightweight structure addition mod messes with the underlying function of forest generation unnecessarily.

**Comment by CREEATION** [comment]
Ah I see. Maybe you are talking about this config option then? _(setting it to false)_

```
# Whether the nature (e.g. trees, mushrooms) added by the mod should be actively generating.
[default: true]
B:generateNature=true
```

**Comment by Avery93** [comment]
If only that were it :'( Despite disabling this config the mod still messes with forest generation, at least in roofed forestes, for some reason: #349

**Comment by CREEATION** [comment]
Nevertheless, this issue seems to be a duplicate of https://github.com/Ivorforce/RecurrentComplex/issues/349.

**Comment by Avery93** [comment]
I would agree if that were the only issue, but it's gone and done something to the vanilla village generation too:
![2019-03-25_01 40 22](https://user-images.githubusercontent.com/48781885/54897513-11fbec00-4e9f-11e9-8d70-b64d712f2cde.png)
Look at all these lights that were added to empty space around a village. Something foundational is changing vanilla world gen rather than just adding to it.

**Comment by CREEATION** [comment]
Just had a look, and could reproduce:

![2019-03-25_10 47 37](https://user-images.githubusercontent.com/3277769/54909889-7c3a7e00-4eeb-11e9-817f-9780a5b4aaec.png)

Not sure why it's generating so many pathways.

**Comment by naqaden** [comment]
I briefly talked to Ivorius on Discord about those lamps. If I understood right, it's from the way vanilla handles the extra potential village structures, even if they are not selected/spawned. I was able to stop generating these lamp paths by removing the village directory of structures from the RC jar, but it should be doable by config alone.

It's too bad, I liked RC's village additions. Maybe they'd be worth allowing to spawn alone randomly in the world like most other structures. I haven't tried.

Similarly, I removed the nature directory of structures (trees + mushrooms) from the jar and Dynamic Trees is working nicely. I anticipate dynamic mushrooms in DT's future.
